docs/understanding-tracking-design/managing-tracking-scenarios-api/index.md

```diff
@@ -0,0 +1,367 @@
+- If **SchemaUndecidable**, a warning will be attached to the `errors` response field.
+
+## Editing a Tracking Scenario
+
+Use this request to edit a tracking scenario within an organization. The [request and response](#request-and-response) follow the same format as during creation.
+
+`PUT /api/msc/v1/organizations/{organizationId}/tracking-scenarios/v1/{trackingScenarioId}`
+
+### Publishing a Tracking Scenario
+
+By default, when a tracking scenario is created and no value is provided for `scenario.status`, it is set to **draft**. This suggests a tracking scenario lifecycle that we recommend following, especially when using the API in conjunction with the Console.
+
+```mermaid
+graph LR
+    style Start fill:#633EB5, stroke:#000000, stroke-width:0px;
+    style Draft color:#724512, fill:#FEEEBD, stroke:#724512, stroke-width:1px; 
+    style Published color:#3F2874, fill:#F0EBF8, stroke:#3F2874, stroke-width:1px;
+    style Deprecated color:#89251F, fill:#FDF3F2, stroke:#89251F, stroke-width:1px;
+    style Deleted fill:#D63A31, stroke:#000000, stroke-width:1px;
+    linkStyle default stroke-width:2px,fill:#F2F4F7,stroke:#633EB5,color:#633EB5
+
+    Start(( )) -->|Create| Draft
+    Draft -->|Publish| Published
+    Draft -->|Delete| Deleted
+    Published -->|Deprecate| Deprecated
+    Deprecated -->|Undeprecate| Draft
+    Deleted((Deleted))
+```
+
+In addition to this lifecycle and in conjunction with versioning, we enforce that when a tracking scenario is **published**, the versions between two published versions are **squashed**. For example:
+
+Publish new version, before squash:
+
+```mermaid
+graph LR
+    style A color:#3F2874, fill:#F0EBF8, stroke:#3F2874, stroke-width:1px;
+    style B color:#724512, fill:#FEEEBD, stroke:#724512, stroke-width:1px;
+    style C color:#724512, fill:#FEEEBD, stroke:#724512, stroke-width:1px;
+    style D color:#724512, fill:#FEEEBD, stroke:#724512, stroke-width:1px;
+    style E color:#3F2874, fill:#F0EBF8, stroke:#3F2874, stroke-width:1px;
+    linkStyle default stroke-width:2px,fill:#F2F4F7,stroke:#633EB5,color:#633EB5
+
+    A[Published 1] --> B[Draft 2]
+    B[Draft 2] --> C[Draft 3]
+    C[Draft 3] --> D[Draft 4]
+    D[Draft 4] --> E[Published 5]
+```
+
+After squash:
+
+```mermaid
+graph LR
+    style A color:#3F2874, fill:#F0EBF8, stroke:#3F2874, stroke-width:1px;
+    style B color:#3F2874, fill:#F0EBF8, stroke:#3F2874, stroke-width:1px;
+    linkStyle default stroke-width:2px,fill:#F2F4F7,stroke:#633EB5,color:#633EB5
+
+    A[Published 1] --> B[Published 5]
+```
+
+### Deprecating a Tracking Scenario
+
+Tracking scenarios can also be **deprecated** by changing the `scenario.status` field to `deprecated`.
```

Comment:
Perhaps worth adding a sentence to explain that this is essentially documentation, so deprecating a scenario is a flag to tell consumers not to rely on this scenario anymore.

Reply:
I elaborated a little bit more what is the intention of 'deprecated' status